### PR TITLE
Improve doc test coverage (A-Base)

### PR DESCRIPTION
**Pull request by mlopatkin**
Part of #21027.

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/authoring-builds/ant.adoc [hunk 1]
```diff
@@ -84,8 +84,8 @@ To make custom tasks available in your build, you can use the `taskdef` (usually
 
 .Using a custom Ant task
 ====
-include::sample[dir="snippets/ant/useExternalAntTask/groovy",files="build.gradle"]
-include::sample[dir="snippets/ant/useExternalAntTask/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/ant/useExternalAntTask/groovy",files="build.gradle[tags=configure-task]"]
```
Comment (6hundreds) [inline subprojects/docs/src/docs/userguide/authoring-builds/ant.adoc:87]: @mlopatkin Btw what the purpose for this change?

**Review by 6hundreds — APPROVED** [review]
LGTM

File: subprojects/docs/src/docs/userguide/authoring-builds/ant.adoc (hunk 1 above)
Comment (mlopatkin) [reply subprojects/docs/src/docs/userguide/authoring-builds/ant.adoc:87]: This is the [asciidoc syntax](https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/) to include only a part of the file. If you look at the changes of `snippets/ant/useExternalAntTask/groovy/build.gradle` you can see that I added some scaffolding code to resolve checkstyle binaries. This new code shouldn't be included in the userguide, so I wrapped the interesting part into tags and added the tag name here.

**Comment by mlopatkin** [comment]
@bot-gradle test BD

**Comment by mlopatkin** [comment]
@bot-gradle test and merge

**Pull request merged**